reconcile/test/test_typed_queries/cost_report/test_get_cost_namespaces.py

```diff
@@ -82,7 +82,7 @@ def test_get_cost_namespaces(
     namespaces = get_cost_namespaces(gql_api)
 
     assert namespaces == [expected_cost_namespace]
-    gql_api.query.assert_called_once_with(
+    gql_api.query.assert_called_once_with(  # type: ignore[attr-defined]
```

Comment:
can we change `gql_api_builder` return type to `MagicMock`? `GqlApi` type will be lost, I wish this is some type like `MagicMock[GqlApi]`.
same for other `# type: ignore[attr-defined]`